smdebug/core/tfevent/util.py

```diff
@@ -10,6 +11,9 @@
 
 logger = get_logger()
 
+# TF Implements a Custom Numpy Datatype for Brain Floating Type
+_np_bfloat16 = _pywrap_bfloat16.TF_bfloat16_type()
```

Comment:
is there a specific requirement for this?

Reply:
There is no special ask for this, but the goal is to support every possible TF type.